test/e2e-v2/cases/redis/redis-exporter/redis.conf

```diff
@@ -54,8 +54,8 @@ auto-aof-rewrite-min-size 64mb
 aof-load-truncated yes
 aof-use-rdb-preamble yes
 lua-time-limit 5000
-slowlog-log-slower-than 10000
-slowlog-max-len 128
+slowlog-log-slower-than 1000
+slowlog-max-len 1200
```

Comment:
I think we should mention this in the docs to guide end users.

Reply:
Got it.